module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -69,14 +76,24 @@ A vignette on the usage of the `ggplot2` package can be found [**here**](https:/
 | Library/Package                      | Piece of code                                   | What it's called       | What it does                                                                                                                                                                                      |
 |--------------------------------------|-----------------------------------------------------|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
 | `ggplot2`         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
+| `ggplot2`               | [`xlab()`; `ylab()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/labs)           | X Axis Labels; Y Axis Labels                    | Modifies the labels on the x axis and on the y axis, respectively        |             
 | `ggplot2`         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when combined with ggplot()            |
```

Comment:
Could you add `geom_point()` and `geom_line()` as those are pretty general use? 

Also, why don't we move all the geoms up to directly below `ggplot()`